packages/plugin-dts/src/apiExtractor.ts

```diff
@@ -5,23 +5,34 @@ import {
   type ExtractorResult,
 } from '@microsoft/api-extractor';
 import { logger } from '@rsbuild/core';
+import color from 'picocolors';
+import { getTimeCost } from './utils';
 
 export type BundleOptions = {
+  name: string;
   cwd: string;
   outDir: string;
+  dtsExtension: string;
   entry?: string;
   tsconfigPath?: string;
 };
 
-export function bundleDts(options: BundleOptions): void {
+export async function bundleDts(options: BundleOptions): Promise<void> {
   const {
+    name,
     cwd,
     outDir,
+    dtsExtension,
     entry = 'index.d.ts',
     tsconfigPath = 'tsconfig.json',
   } = options;
   try {
-    const untrimmedFilePath = join(cwd, relative(cwd, outDir), 'index.d.ts');
+    const start = Date.now();
+    const untrimmedFilePath = join(
+      cwd,
+      relative(cwd, outDir),
+      `index${dtsExtension}`,
```

Comment:
The `index` is fixed for now?

Reply:
What should our bundled DTS file named

Reply:
Provide naming config to user, we could implement this in the future.